crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -16,13 +17,13 @@ use starknet::signers::SigningKey;
 
 #[derive(Args, Debug)]
 #[command(about = "Add an account to the accounts file")]
-pub struct Add {
-    /// Name of the account to be added
+pub struct Import {
+    /// Name of the account to be imported
     #[clap(short, long)]
     pub name: String,
 
     /// Address of the account
-    #[clap(short, long, requires = "private_key_input")]
+    #[clap(short, long)]
     pub address: Felt,
```

Comment:
note: `private_key_input` is not required for `address` now because user will be asked for private key  in interactive mode if it's not passed as arg.